winui-toc.html

```diff
@@ -375,6 +375,7 @@
         <ul>
             <li> <a href="/winui/Kanban/overview">Overview</a></li>
             <li><a href="/winui/Kanban/getting-started">Getting Started</a></li>
+            <li><a href="/winui/Kanban/columns">Columns</a></li>
```

Comment:
Follow this order and resolve conflict error

- Overview
- Getting Started
- Column
- Cards

Reply:
File order changed